pkg/kind/kind.go

```diff
@@ -26,7 +26,7 @@ import (
 	"knative.dev/kn-plugin-quickstart/pkg/install"
 )
 
-var kubernetesVersion = "v1.22.4"
+var kubernetesVersion = "v1.23.3"
```

Comment:
Just making a note that latest version as of today for kind node image `kindest/node` is https://hub.docker.com/layers/kindest/node/v1.23.3/images/sha256-0df8215895129c0d3221cda19847d1296c4f29ec93487339149333bd9d899e5a?context=explore